python/cudf/cudf/utils/applyutils.py

```diff
@@ -111,12 +111,27 @@ def apply_chunks(
 
 
 def make_aggregate_nullmask(df, columns=None, op="and"):
+    from cudf import MultiIndex
```

Comment:
Don't import in the function, just use `cudf.MultiIndex` below

Reply:
cudf isn't imported in the file, so I might have to import cudf before that.